### DISCUSSION TITLE
SIGNUPS_ALLOWED env variable not working

### DISCUSSION THREAD

**Discussion by mlewis26201** [dani-garcia/vaultwarden#2659] [Q&A] [answered]
I have SIGNUPS_ALLOWED=false set and it is still allowing new users to sign up. Tried recreating the container and no change. What can I check the determine the issue?

**Comment by BlackDex** [comment, marked as answer]
You probably have a `config.json` in your data directory.
Please configure this setting via the `/admin` interface.

**Reply by mlewis26201** [reply]
This worked, but I am not sure how the config.json got there unless it is created when you access the admin interface or something. I know I did not manually create it.

**Reply by BlackDex** [reply]
Of you ever used the admin interface too change settings before too change settings it creates that file and overrules all env values.
